io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/reader/XLSInstanceReader.java

```diff
@@ -108,23 +105,71 @@ protected IOReport execute(ProgressIndicator progress, IOReporter reporter)
 		propAr = type.getChildren().toArray(new PropertyDefinition[type.getChildren().size()]);
 		Collection<List<String>> rows = analyser.getRows();
 
-		// skip if first row is a header
-		if (!skipFirst) {
-			// otherwise first line is also an instance
+		int skipNlines = 0;
+		Boolean skipType = getParameter(CommonSchemaConstants.PARAM_SKIP_N_LINES).as(Boolean.class);
+
+		if (skipType == null) {
+
+			skipNlines = getParameter(CommonSchemaConstants.PARAM_SKIP_N_LINES).as(Integer.class);
+
+		}
+		else if (skipType) {
+
+			skipNlines = 1;
+
+		}
+		else {
+
+			skipNlines = 0;
+		}
+
+		// do not skip any lines
+		if (skipNlines == 0) {
+
+			// also the header is retrieved
 			createInstanceCollection(analyser.getHeader(), reporter);
+
 			line++;
+
+			readRows(reporter, rows);
```

Comment:
Looks like after the modification of converting `boolean` to `int`, `readRows(reporter, rows);` is used only in one place. I believe that the new method can be removed and its code can be pasted here.